CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Unreleased
 
+IMPROVEMENTS:
```

Comment:
I think this might need to be listed as a breaking change too? Since doing a full restart of all the clients and servers can be destructive.

Reply:
Sure thing.

Reply:
done 👍